geopy/distance.py

```diff
@@ -120,6 +119,39 @@
     'GRS-67':        (6378.1600, 6356.774719, 1 / 298.25)
 }
 
+
+def lonlat(x, y, z=0):
+    """
+    Helper function that can be used for more explicit and obvious calling of the distance methods
```

Comment:
I don't think that `obvious` is a proper term for documentation :-)

This description should introduce the reader to the problem: `distance` accepts coordinates in `(y, x)`/`(lat, lon)` order, while some other libraries/systems might use `(x, y)`/`(lon, lat)` instead.

And then suggest a solution: using this function, which accepts coordinates in `(x, y, [z])` order and returns a Point instance for the `distance`.